UniversalScanner.sln

```diff
@@ -3,35 +3,85 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.29519.181
 MinimumVisualStudioVersion = 10.0.40219.1
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UniversalScannerLib", "UniversalScannerLib\UniversalScannerLib.csproj", "{5EAB95CA-2F8B-480A-A6C6-0CE59DB65DE3}"
+EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UniversalScanner", "UniversalScanner\UniversalScanner.csproj", "{6A0EAAE3-897B-4472-A692-CA0B08E09CD1}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "selftest2pcap", "selftest2pcap\selftest2pcap.csproj", "{6127A446-FAC5-4A53-8F0A-01C32F657DFF}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "selftest2pcap", "selftest2pcap\selftest2pcap.csproj", "{039F6A97-4389-430B-8DBC-242C1E682F17}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "scannerTemplate", "scannerTemplate\scannerTemplate.csproj", "{CA2EBD82-FF40-4A5B-AEC6-D95104D159EF}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "scannerTemplate", "scannerTemplate\scannerTemplate.csproj", "{32E255E4-8C44-470F-911A-E29729DCFA8E}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "UniversalScannerService", "UniversalScannerService\UniversalScannerService.csproj", "{E69C2B74-EDC5-4362-9D7D-AA88C93C24D5}"

```

Comment:
Sorry, but I do not see added value to split interface and core for such small tool for now.